README.md

```diff
@@ -1,2 +1,28 @@
+# WARNING
+
+This is WIP repo and it is under development. Use it at your own risk. 
+If you have use-cases for such thing to be developed, please submit
+an issue or PR with description of your needs / fixes.
+
 # meta-trenchboot-demo
+
 Meta layer for Trenchboot related demos
+
+
+# Building
+
+```
+SHELL=bash kas-docker --ssh-dir ~/ssh-keys build meta-trenchboot-demo/kas.yml
+```
+
+Change directory to build/tmp/deploy/images/pcengines-apu2 and flash the drive
+with bmap-tools:
+
+```
+bmaptool copy --bmap xen-dom0-image-pcengines-apu2.wic.bmap xen-dom0-image-pcengines-apu2.wic.gz /dev/<dev-dir>
```

Comment:
@Asiderr if you use parameters `<dev-dir>` then name should be descriptive and described in text. I have no idea what you mean by `<dev-dir>`

Reply:
@pietrushnic I add the description, with instruction how to get that parameter.